provider/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/pulumi/pulumi-random/provider/v4
 go 1.18
 
 require (
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.28.0
-	github.com/pulumi/pulumi/sdk/v3 v3.38.0
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.33.1-0.20221104131001-330791e758b8
```

Comment:
Hold on, this may have been accidental, I did not intend to reference a pre-release version.

Reply:
Ah. We're getting this by updating to pulumi/pkg v3.48.0 indirectly which depends on this version of the bridge.